list-of-databases.txt

```diff
@@ -0,0 +1,12 @@
+                               List of databases
+    Name     |  Owner   | Encoding | Collate |  Ctype  |   Access privileges   
+-------------+----------+----------+---------+---------+-----------------------
+ dbBookLover | dev      | UTF8     | C.UTF-8 | C.UTF-8 | 
+ dev         | dev      | UTF8     | C.UTF-8 | C.UTF-8 | 
+ postgres    | postgres | UTF8     | C.UTF-8 | C.UTF-8 | 
+ template0   | postgres | UTF8     | C.UTF-8 | C.UTF-8 | =c/postgres          +
+             |          |          |         |         | postgres=CTc/postgres
+ template1   | postgres | UTF8     | C.UTF-8 | C.UTF-8 | =c/postgres          +
+             |          |          |         |         | postgres=CTc/postgres
+(5 rows)
```

Comment:
You can remove this file

Reply:
<img width="1039" alt="Screen Shot 2022-09-16 at 10 01 17 AM" src="https://user-images.githubusercontent.com/106217355/190691269-1d8d6348-19ee-4ab2-8954-dd0ce2ebb251.png">